app/models/repository.rb

```diff
@@ -1,3 +1,7 @@
 class Repository < KalibroClient::Entities::Processor::Repository
   include KalibroRecord
+
+  def self.latest(count=1)
+    all.sort { |one, another| another.id <=> one.id }.first(count)
```

Comment:
This will fetch every single repository any time it's called. The sort should be changed to something that ActiveRecord can handle to avoid that.